### PR TITLE
Personal branch

### PR DESCRIPTION
**Pull request by kevisu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,3 @@ We will be experimenting with GitHub PR workflow here.
 
 ## Here is the history of commits
 * This message was made from the personal account - Aug 26th 2020
```
Comment (kevisu) [inline README.md:6]: Adding changes

**Review by kevisu — COMMENTED** [review]
Here is some more info for this comment.

File: README.md [hunk 2]
```diff
@@ -4,4 +4,3 @@ We will be experimenting with GitHub PR workflow here.
 
 ## Here is the history of commits
```
Comment (kevisu) [inline README.md:5]: Why is this here

File: README.md [hunk 3]
```diff
@@ -4,4 +4,3 @@ We will be experimenting with GitHub PR workflow here.
...
 * This message was made from the personal account - Aug 26th 2020
-* This message was made from the work account - Aug 26th 2020
```
Comment (kevisu) [inline README.md:7]: Why was this line deleted?

**Review by kevisu — COMMENTED** [review]
Please add this back

**Comment by kevisu** [comment]
LGTM!

**Review by kevinnsun — APPROVED** [review]
